draft-ietf-quic-transport.md

```diff
@@ -3604,23 +3503,55 @@ The layout of a Version Negotiation packet is:
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                   [Supported Version N (32)]                ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                       Unused Payload (*)                    ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ~~~
 {: #version-negotiation-format title="Version Negotiation Packet"}
 
-The value in the Unused field is selected randomly by the server.
-
 The Version field of a Version Negotiation packet MUST be set to 0x00000000.
+A Version Negotiation packet contains the following fields:
 
-The server MUST include the value from the Source Connection ID field of the
-packet it receives in the Destination Connection ID field.  The value for Source
-Connection ID MUST be copied from the Destination Connection ID of the received
-packet, which is initially randomly selected by a client.  Echoing both
-connection IDs gives clients some assurance that the server received the packet
-and that the Version Negotiation packet was not generated by an off-path
-attacker.
+Reserved Bits (R):
```

Comment:
There is a lot of added text here.  Is that because the long header packets are currently not correctly documented?

Reply:
The added text here is to add new fields to VN packets. This PR adds ODCIL and Original Destination Connection ID to allow the server to send a new source connection ID (similar to how Retry packets work today). The PR also adds a Number of Supported Versions field to allow future extensibility.